x/stakeibc/types/params.go

```diff
@@ -22,6 +22,8 @@ var (
 	DefaultICATimeoutNanos  uint64 = 600000000000
 	DefaultBufferSize       uint64 = 5   // 1/5=20% of the epoch
 	DefaultIbcTimeoutBlocks uint64 = 300 // 300 blocks ~= 30 minutes
+	DefaultFeeTransferTimeoutNanos  uint64 = 600000000000 // 10 minutes
```

Comment:
What scenario are you envisioning might prompt us to change this through governance?

Reply:
just gives us a bit more flexibility, it's possible we find calls are timing out frequently if we only allow for a 10 min buffer (can't think of a reason not to parameterize, although lmk if you're thinking of one)